qt/python/mantidqt/widgets/sliceviewer/view.py

```diff
@@ -11,7 +11,7 @@
 from qtpy.QtWidgets import QWidget, QVBoxLayout
 from qtpy.QtCore import Qt
 from matplotlib.figure import Figure
-from matplotlib.backends.backend_qt5agg import FigureCanvas, NavigationToolbar2QT as NavigationToolbar
+from MPLwidgets import FigureCanvas, NavigationToolbar2QT as NavigationToolbar
```

Comment:
Any chance we could move the functionality in `MPLwidgest` to `mantidqt` as a util to import the matplotlib backend? It'd be nice if `mantidqt` could be standalone.

Reply:
I did think using something from `scripts` wasn't the best thing to do. Do you think I could just move https://github.com/mantidproject/mantid/blob/master/scripts/MPLwidgets.py and https://github.com/mantidproject/mantid/blob/master/scripts/gui_helper.py into `qt/python/mantidqt` and update the imports?

Reply:
I think that would be fine.